www/css/album.css

```diff
@@ -0,0 +1,50 @@
+.album-content {
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.album-info-block{
+  margin-left: 2px
+}
+
+.album-images {
+  position: relative;
+  overflow: hidden;
+  height: calc(25vw - 11.5px);
+  width: calc(25vw - 11.5px);
+  float: left;
+  margin-left: 2px;
+  margin-right: 2px;
+  margin-bottom: 4px;
```

Comment:
Group margins and remove overflow

Reply:
I'm not sure the element will float if the overflow is removed here: [http://webdesignerwall.com/tutorials/css-clearing-floats-with-overflow](http://webdesignerwall.com/tutorials/css-clearing-floats-with-overflow)